dev-cpp/tbb/patches/tbb-2022.0.0.patchset

```diff
@@ -0,0 +1,230 @@
+From 53cc4fc65dac5437f6d6197e278b3434e43ab594 Mon Sep 17 00:00:00 2001
+From: Schrijvers Luc <begasus@gmail.com>
+Date: Mon, 25 Nov 2024 17:46:22 +0100
+Subject: Haiku patches
+
+
+diff --git a/cmake/compilers/Clang.cmake b/cmake/compilers/Clang.cmake
+index dcd6663..98812d6 100644
+--- a/cmake/compilers/Clang.cmake
++++ b/cmake/compilers/Clang.cmake
+@@ -70,10 +70,12 @@ if (NOT APPLE)
+     set(TBB_LIB_LINK_FLAGS ${TBB_LIB_LINK_FLAGS} -Wl,-z,relro,-z,now)
+ endif()
+ 
+-set(TBB_COMMON_LINK_LIBS ${CMAKE_DL_LIBS})
++if (NOT HAIKU)
++    set(TBB_COMMON_LINK_LIBS ${CMAKE_DL_LIBS})
```

Comment:
Isn't `${CMAKE_DL_LIBS}` empty on Haiku anyway?

Reply:
IIRC this one failed not finding ldl

Reply:
This looks like a bug in CMake to me then. The variable exists exactly for the purpose to abstract OS differences. It should be empty on Haiku and not equal to "dl".